core-tests/shared/src/test/scala/zio/DurationSpec.scala

```diff
@@ -154,7 +154,7 @@ object DurationSpec extends ZIOBaseSpec {
         assert(Duration.Infinity.isZero)(equalTo(false))
       },
       test("It converts into the infinite s.c.d.Duration") {
-        assert(Duration.Infinity.asScala)(equalTo(ScalaDuration.Inf: ScalaDuration))
+        assert(Duration.Infinity.asScala)(equalTo(ScalaFiniteDuration(Long.MaxValue, TimeUnit.NANOSECONDS)))
```

Comment:
I will note this is likely not preferable if users are using APIs which special case for `Duration.Inf`...